spring-kafka/src/main/java/org/springframework/kafka/support/KafkaUtils.java

```diff
@@ -33,6 +35,12 @@
  */
 public final class KafkaUtils {
 
+	/**
+	 * True if micrometer is on the class path.
+	 */
+	public static final boolean MICROMETER_PRESENT = ClassUtils.isPresent(
+			"io.micrometer.core.instrument.MeterRegistry", KafkaMessageListenerContainer.class.getClassLoader());
```

Comment:
I think this is copy/paste artifact.
Or `KafkaUtils` or `ClassUtils.getDefaultClassLoader()`